library/waku_thread/inter_thread_communication/requests/node_lifecycle_request.nim

```diff
@@ -63,11 +63,32 @@ proc createNode(configJson: cstring): Future[Result[WakuNode, string]] {.async.}
     return err("exception calling parseConfig: " & getCurrentExceptionMsg())
 
   # TODO: figure out how to extract default values from the config pragma
-  conf.clusterId = 0
   conf.nat = "any"
   conf.maxConnections = 50.uint16
   conf.maxMessageSize = default_values.DefaultMaxWakuMessageSizeStr
 
+  # The Waku Network config (cluster-id=1)
+  if conf.clusterId == 1:
+    let twnClusterConf = ClusterConf.TheWakuNetworkConf()
+    if len(conf.shards) != 0:
+      conf.pubsubTopics = conf.shards.mapIt(twnClusterConf.pubsubTopics[it.uint16])
+    else:
+      conf.pubsubTopics = twnClusterConf.pubsubTopics
+
+    # Override configuration
+    conf.maxMessageSize = twnClusterConf.maxMessageSize
+    conf.clusterId = twnClusterConf.clusterId
+    conf.rlnRelay = twnClusterConf.rlnRelay
+    conf.rlnRelayEthContractAddress = twnClusterConf.rlnRelayEthContractAddress
+    conf.rlnRelayDynamic = twnClusterConf.rlnRelayDynamic
+    conf.rlnRelayBandwidthThreshold = twnClusterConf.rlnRelayBandwidthThreshold
+    conf.discv5Discovery = twnClusterConf.discv5Discovery
+    conf.discv5BootstrapNodes =
+      conf.discv5BootstrapNodes & twnClusterConf.discv5BootstrapNodes
+    conf.rlnEpochSizeSec = twnClusterConf.rlnEpochSizeSec
+    conf.rlnRelayUserMessageLimit = twnClusterConf.rlnRelayUserMessageLimit
```

Comment:
We have a similar structure in https://github.com/waku-org/nwaku/blob/18a053591d8d18b8f066d4bc919c3910cb4295c3/apps/wakunode2/wakunode2.nim#L91-L109

I wonder if we could encapsulate that somehow. Maybe in a separate PR